lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -41,8 +41,11 @@ defmodule Mix.Tasks.Xref do
 
     * `--label` - only shows relationships with the given label.
       By default, it keeps all labels that are transitive.
-      The labels are "compile", "export" and "runtime". See
-      "Dependencies types" section below
+      The labels are "compile", "export" and "runtime".
+      Use "compile-connected" to restrict the relationships that
+      are compile-time *and* towards files that themselves have
+      other dependencies of any type.
+      See "Dependencies types" section below.
```

Comment:
```suggestion
      The label modifier "compile-connected" can be used to
      find modules that have at least one compile dependency
      between them, excluding the compile time dependencies
      themselves. See "Dependencies types" section below.
```

Reply:
I am merging this revision of the docs but it will likely change again once I remove only_direct, if you have fixes or suggestions, they are welcome! :heart: